### ISSUE TITLE
Plugin maintenance

### ISSUE THREAD

**Issue by slotthhy** [ff-meli/OrchestrionPlugin#19]
Hi, I know you've been pretty busy and I don't blame you if plugins are the last thing on your mind. A lot of people like Orchestrion and ask questions about it, and I'd guess a number of people contact you directly about it as well.

Would you mind if I took over development of OrchestrionPlugin for the foreseeable future? I feel like it would be in everyone's best interest for the time being. No hard feelings if not.

**Comment by ff-meli** [comment]
@lmcintyre yeah, that is probably for the best at this point, since I have no idea when/if I'd have time for proper dev work on this anymore.  Ideally I'd like to keep it and continue working on it, but realistically that hasn't happened and it's been stagnating for too long.

Please keep my name somewhere in the app/credits/etc as the original creator, but beyond that go ahead with whatever you want to do with it.

Point goat over here if he's wary of letting you contribute over the plugin repo version. (Hi goat!)

(Unrelated dev note - if penumbra ever gets further along, which at least it looks like it's finally gotten some attention just recently, it should make inclusion of actual orchestrion tracks pretty simple.  The main issue in the current plugin is that orch playback goes through a separate path that uses 3d positional sound, which doesn't work if you're not in a tiny enclosed space.  But swapping the actual asset path of a bgm track to an orch track at runtime will play the orch track through the normal bgm system and seemed to work fine.. I just never finished the RE work to do that in the existing system, and penumbra was going to make it all moot anyway so I never felt like following up.  That was the one big feature I always felt was silly that it was missing... especially given the plugin name >.> )

**Comment by slotthhy** [comment]
I wouldn't even think of removing you as the creator or removing your credits. I have had a few ideas lately so I will probably try to get some stuff done soon and put out an update. Just ask me whenever you want development back and I'll head on out.

Thank you!
